src/SmartComponents/Order/Orders.js

```diff
@@ -26,7 +26,7 @@ class Orders extends Component {
 
   componentDidMount() {
     this.props.fetchOrders()
-    .then(() => this.setState({ orders: this.props.orderList.items }));
+    .then(() => this.setState({ orders: this.props.orderList.items.sort((a, b) => new Date(b.created_at) - new Date(a.created_at)) }));
```

Comment:
Can we use `id` (converted to an integer) for sorting?  Seems it would be less overhead than converting the date for each object.